### PR TITLE
pighead: new, 20201028

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

`pighead NULL => 20201028`

Package(s) Affected
-------------------

pighead

Security Update?
----------------

No

Architectural Progress
----------------------

    

- [x] Architecture-independent `noarch`

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [ ] Architecture-independent `noarch`

### CODE REVIEW

**Comment by OriginCode** [comment]
LFGTM!
image

File: extra-misc/pighead/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,2 @@
+abinfo "Installing Pighead (Gumblex)..."
+install -Dm755 "$SRCDIR"/pighead "$PKGDIR"/usr/bin/pighead
```
Comment (chenx97) [inline extra-misc/pighead/autobuild/build]: Make it verbose, i.e. `-Dvm755`, please.

**Review by chenx97 — CHANGES_REQUESTED** [review]
Changes requested.

**Comment by OriginCode** [comment]
Change version, close PR, a new PR will be opened later.

**Pull request closed without merging**